tests/Functional/ItemsTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentApiBundle\Functional;
+
+use Symfony\Component\HttpFoundation\Request;
+
+final class ItemsTest extends FunctionalTestCase
+{
+    /**
+     * @test
+     * @dataProvider serviceProvider
+     */
+    public function it_pings(string $prefix) : void
+    {
+        $kernel = static::getKernel('Basic');
+
+        $request = Request::create("/{$prefix}/ping");
+
+        $response = $kernel->handle($request);
+
+        $this->assertSame('pong', $response->getContent());
+    }
+
+    public function serviceProvider() : iterable
+    {
+        yield 'foo' => ['foo'];
```

Comment:
more realistic values than `foo` and `bar` help people focusing on what these values should be substituted with e.g. `research-articles`, `green-widgets`, `blog-articles`